src/numberMask/index.js

```diff
@@ -1,20 +1,15 @@
-import * as R from 'ramda';
-
-// Removes non numeric values
-const removeNonNumeric = value => value.replace(/[^0-9.]+/g, '');
-
-const addCommasToNumber = (value = '') => value.replace(/(\d)(?=(\d\d\d)+(?!\d))/g, '$1,');
-
+/**
+ * Given a Number, return a masked version with commas and squash to 2 fractionals
+ * @param {Number} value Number to be masked
+ * @returns String
+ */
 const numberMask = (value) => {
-  const stringValue = String(value);
-  const isNegative = stringValue.charAt(0) === '-';
+  if (!value) return '';
+  if (value === '-') return '-';
+  const valueAsNumber = parseFloat(value);
```

Comment:
Problem with parseFlow/parseInt is that if a string is passed in by mistake the parsing will end at the first none numerical character, e.g `9,999` would be parsed as `9`

Reply:
We could cast the input parameter to a string and then try and do a regex replace of [non numeric, '.', '-'], if we did this we would need the same previous regex used to track for negative numbers I think ```const numberValue = isNegative ? `-${removeNonNumeric(R.tail(stringValue))}` : removeNonNumeric(stringValue);``` due to lack of browser compatibility with negative look behind regex

Reply:
If we just tried to remove all non numeric values first we would run the risk of modifying a string that gets passed in such as "-1" and returning "1"